Automattic/crowdsignal-ui
# Update Share Modal to include Embed Popup settings

## Summary

This PR complements #311 and includes the necessary changes to allow the user to customize the Embed Popup settings through the Share Modal

Depends On: D86900-code

Fixes Automattic/crowdsignal#452

![image](https://user-images.githubusercontent.com/7811225/187561329-242fdf91-14ee-41e1-9e6e-118413063914.png)

## Test Instructions

* Apply backend changes
* Open or create a project and make sure it's published
* Click the Share button
* You should be able to see the `Edit Appearance` button on the Embed Popup card
* Click the button and play with the settings
* The settings should be saved automatically as soon as you stop editing
